src/ontology/mondo-edit.obo

```diff
@@ -505014,8 +505018,8 @@ xref: OMIM:620040 {source="MONDO:equivalentTo"}
 xref: UMLS:C5774217 {source="MEDGEN:1823990", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0015780 {source="OMIM:620040"} ! dyskeratosis congenita
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12441 {source="OMIM:620040"} ! TYMS
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/30365 {source="OMIM:620040"} ! ENOSF1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12441 ! TYMS
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/30365 ! ENOSF1
```

Comment:
The disease description contains "digenic" for https://omim.org/entry/620040 and these gene associations should be removed.

Reply:
See: [response](https://github.com/monarch-initiative/mondo/pull/8624#discussion_r1933109660)

Reply:
This is in the [exclusion file](https://github.com/monarch-initiative/omim/blob/main/data/exclusions-disease-gene.tsv) and should be removed.